src/rules/rust_style.rs

```diff
@@ -0,0 +1,1922 @@
+// Copyright 2018-2024 the Deno authors + c-antin. All rights reserved. MIT license.
+// based on camelcase.rs
```

Comment:
compare first commit with last commit for useful diff, because `rust_style.rs` is based on `camelcase.rs`